### PR TITLE
Make response time of the example server more interesting.

### PR DESCRIPTION
**Pull request by beorn7**
Make sigma depend on the average latency.

Create periodic "outages" with `10*error_ratio` and `3*latency`.

Without the latter, latency graphs will just by flat lines.

@brian-brazil We'll conduct a workshop at each of the upcoming velocities. So I thought it's a good time to spice this up.

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -25,10 +28,10 @@ def handler_404(self):
 ROUTES = {
     ('GET', "/"): lambda self: self.wfile.write("Hello World!"),
     ('GET', "/favicon.ico"): lambda self: self.send_response(404),
-    ('GET', "/api/foo"): generate_request_handler(.01, .005),
-    ('POST', "/api/foo"): generate_request_handler(.02, .02),
-    ('GET', "/api/bar"): generate_request_handler(.015, .00025),
-    ('POST', "/api/bar"): generate_request_handler(.05, .01),
+    ('GET', "/api/foo"): generate_request_handler(.01, .005, 23.),
```
Comment (brian-brazil) [inline server.py]: I'd do 23.0, just 23. seems weird to me

Comment (beorn7) [inline server.py]: Sure.

**Comment by brian-brazil** [comment]
We may want to move these repos into somewhere more central

**Comment by beorn7** [comment]
Yup. For now, I'll copy it over into https://github.com/juliusv/prometheus_workshop so that people have it all in one place for the upcoming workshops. We'll see how it goes and can put it up at a more fitting place eventually.

**Pull request merged**